rest-api/src/main/java/uk/gov/hmcts/ccd/definition/store/rest/endpoint/UserRoleController.java

```diff
@@ -67,4 +67,13 @@ UserRole userRoleGet(@RequestParam("role") @NotNull byte[] roleBase64EncodedByte
         @ApiParam(value = "Roles", required = true) @PathVariable("roles") List<String> roles) {
         return this.userRoleService.getRoles(roles);
     }
+
+    @RequestMapping(value = "/all-roles", method = RequestMethod.GET, produces = {"application/json"})
```

Comment:
Can we change the Mapping to `/user-roles` to make the naming consistent with other endpoints? I think we do not need to mention all explicitly so leaving it out would not obscure readability.